CHANGELOG.md

```diff
@@ -14,42 +14,53 @@ Bug Fixes
+- Idle Track Switch Off uses document visibility, track attachments, and the visibility of video elements to determine whether a RemoteVideoTrack should be switched off. A RemoteVideoTrack will be switched off when the document is no longer visible, no video elements are attached to the track, or when the video elements attached to the track are not visible.
 
-```js
-  const { connect } = require('twilio-video');
+**Auto renderDimensions**
 
-  const room = await connect(token, {
-    name: "my-new-room",
-    bandwidthProfile: {
-      video: {
-        idleTrackSwitchOff: true,
-      }
-    }
-  });
-```
+- The SDK now uses the dimensions of the video elements attached to a RemoteVideoTrack to determine the best video bitrate to receive. A RemoteVideoTrack attached to a video element with larger dimensions will get a higher quality video compared to a RemoteVideoTrack attached to a video renderer with smaller dimensions.
 
-Note: This feature relies on applications using the `attach` and `detach` methods of a RemoteVideoTrack. If your application currently uses the underlying MediaStreamTrack to associate RemoteVideoTracks to video elements, you will need to update your application to use those methods. This feature can be disabled by setting `idleTrackSwitchOff` property to false in the VideoBandwidthProfileOptions dictionary.
 
-```js
-  const { connect } = require('twilio-video');
+Both these features are available in Group Rooms and are enabled by default if your application specifies [Bandwidth Profile options](https://media.twiliocdn.com/sdk/js/video/releases/2.12.0/docs/global.html#BandwidthProfileOptions__anchor) during connect. If your application previously set legacy `renderDimensions` you must change the value to `"auto"` in order to take advantage of automatic hinting.
 
-  const room = await connect(token, {
-    name: "my-new-room",
-    bandwidthProfile: {
-      video: {
-        idleTrackSwitchOff: false,
+  ```js
+    const { connect } = require('twilio-video');
+
+    const room = await connect(token, {
+      name: "my-new-room",
+      bandwidthProfile: {
+        video: {
+          idleTrackSwitchOff: true,
+          renderDimensions: "auto",
+        }
       }
-    }
-  });
-```
+    });
+  ```
 
+Note: These features rely on applications using [attach](https://media.twiliocdn.com/sdk/js/video/releases/2.12.0/docs/RemoteVideoTrack.html#attach__anchor) and [detach](https://media.twiliocdn.com/sdk/js/video/releases/2.12.0/docs/RemoteVideoTrack.html#detach__anchor) methods of `RemoteVideoTrack`. If your application currently uses the underlying `MediaStreamTrack` to associate Tracks to video elements, you will need to update your application to use the attach/detach methods. `idleTrackSwitchOff` can be disabled by specifying `false` for the field in BandwidthProfileOptions dictionary. You can also specify explicit legacy renderDimensions based on priority instead of video element sizes.
```

Comment:
```suggestion
Note: These features rely on applications using [attach](https://media.twiliocdn.com/sdk/js/video/releases/2.13.1/docs/RemoteVideoTrack.html#attach__anchor) and [detach](https://media.twiliocdn.com/sdk/js/video/releases/2.13.1/docs/RemoteVideoTrack.html#detach__anchor) methods of `RemoteVideoTrack`. If your application currently uses the underlying `MediaStreamTrack` to associate Tracks to video elements, you will need to update your application to use the attach/detach methods. `idleTrackSwitchOff` can be disabled by specifying `false` for the property in the VideoBandwidthProfileOptions dictionary. You can also specify explicit legacy `renderDimensions` if you want to allocate bandwidth based on Track priority instead of the dimensions of the attached video element(s).
```